packages/theme-cart/README.md

```diff
@@ -2,6 +2,16 @@
 
 Theme Cart is a tiny library (<1kb min+gzip) that facilitates requests to [Shopify's Cart API](https://help.shopify.com/en/themes/development/getting-started/using-ajax-api) and makes it easier to manage cart state.
 
+## Browser Support
+
+This library is compatible with the following browsers:
+
+| Chrome | Edge | Firefox | IE  | Opera | Safari |
+| ------ | ---- | ------- | --- | ----- | ------ |
+| ✓      | ✓    | ✓       | ✕*  | ✓     | ✓      |
+
+* The script is no longer compatible with IE11 starting v3.0.0. The latest version that supports IE11 is v2.0.2.
```

Comment:
Actually. We will need to change this from `v3.0.0` to `v4.0.0`. I forgot that the version bump is based on the latest `theme-script` release version - meaning all packages get updated to the latest version. Since theme-cart is currently `3.1.0` on npm, this will be updated to `v4.0.0` with the breaking change.